framework/db/mapping_manager.py

```diff
@@ -74,6 +74,13 @@ def GetMappingTypes(self):
         """
         return self.mapping_types
 
+    def GetAllMappings(self):
```

Comment:
Function names should always be in snake case according to PEP specifications. I am changing every other names when refactoring.